Sample/Kitchen Sink/DemonstrationTools/DemonstrationViewModel.swift

```diff
@@ -0,0 +1,122 @@
+//
+//  DemonstrationViewModel.swift
+//  Kitchen Sink
+//
+//  Created by Jake Dove on 11/03/2024.
+//
+
+import Foundation
+import Combine
+import FlightUI
+import SwiftUI
+
+ class DemonstrationViewModel : ObservableObject {
+
+    @Published var kgInputString: String = ""
+    @Published var lbsInputString: String = ""
+    @Published var emptyFields = false
+    @Published var inputValue: String = ""
+    @Published var inputUnit: LengthType = .feet
+    @Published var outputUnit: LengthType = .metres
+    @Published var outputValue: String = ""
+    @Published var boundSelectionInput : LengthType? = .feet
+    @Published var boundSelectionOutput : LengthType? = .metres
+
+     private let feetToMetresConversionRate: Decimal = 3.28084
```

Comment:
Indentation of these looks different to the rest 🤔

Reply:
Corrected